File: kotlin/lib/src/main/kotlin/models/BackgroundTaskStatus.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import kotlinx.serialization.json.Json
import kotlinx.serialization.json.encodeToJsonElement
import kotlinx.serialization.json.jsonPrimitive

@Serializable
public enum class BackgroundTaskStatus {
    @SerialName("running") RUNNING,
    @SerialName("finished") FINISHED,
    @SerialName("failed") FAILED;

    override fun toString(): String = Json.encodeToJsonElement(this).jsonPrimitive.content
}

File: javascript/src/models/background_task_status.ts
// this file is @generated

export enum BackgroundTaskStatus {
  Running = "running",
  Finished = "finished",
  Failed = "failed",
}

export namespace BackgroundTaskStatusUtil {
  export function _fromJsonObject(object: any): BackgroundTaskStatus {
    return object;
  }

  export function _toJsonObject(self: BackgroundTaskStatus): any {
    return self;
  }
}
